incubator/binding-kafka-grpc/src/main/java/io/aklivity/zilla/runtime/binding/kafka/grpc/internal/KafkaGrpcBinding.java

```diff
@@ -15,15 +15,21 @@
 package io.aklivity.zilla.runtime.binding.kafka.grpc.internal;
 
 import java.net.URL;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+import java.util.concurrent.atomic.AtomicInteger;
 
 import io.aklivity.zilla.runtime.engine.EngineContext;
 import io.aklivity.zilla.runtime.engine.binding.Binding;
 import io.aklivity.zilla.runtime.engine.config.KindConfig;
 
 public final class KafkaGrpcBinding implements Binding
 {
+    private static final int INITIAL_WORKER_COUNT = 0;
     public static final String NAME = "kafka-grpc";
 
+    private final ConcurrentMap<Long, AtomicInteger> workers = new ConcurrentHashMap<>();
```

Comment:
Please assign this in the constructor instead.
We typically only assign constants inline, or non-static shared objects that are instance-scoped constants.